code/_helpers/lists.dm

```diff
@@ -699,8 +699,6 @@ proc/dd_sortedTextList(list/incoming)
 		return 0
 	return L[A.type]
 
-#define listequal(A, B) (A.len == B.len && !length(A^B))
```

Comment:
did this listequal function even work? I'm not seeing how it would compare list contents
have you removed it or just moved it?

Reply:
Removed. 
Comparations was here:
 `A^B` 
If A and B are lists, the result is a list containing items that are in either list but not both. list(1,2) ^ list(2,3) is equivalent to list(1,3). The items found only in A come first in the result, followed by any items found only in B.